gapis/api/vulkan/find_issues.go

```diff
@@ -286,18 +284,19 @@ func (t *findIssues) Flush(ctx context.Context, out transform.Writer) error {
 			var issue replay.Issue
 			msg := eMsg.GetMsg()
 			label := eMsg.GetLabel()
-			if int(label) < t.numInitialCmds {
+			issue.Command = api.CmdID(label)
+			issue.Severity = service.Severity(uint32(eMsg.GetSeverity()))
+
+			if label == 0 {
```

Comment:
Should this be `int(label) == 0` ?

Reply:
No, previously it was casted into int becausethe type of t.numInitialCmds were int. Currently you don't need it. It's original type is uint64 and it is better to compare as it is.